tests/helpers/http.ts

```
@@ -0,0 +1,29 @@
+
+import {BASE_URL} from '@sx/utils/http'
+
+
+/**
+ * An HTTP client owned by the test, plus an adapter bound to it.
+ *
+ * Services and resources now make requests through an injected client rather than the global axios
+ * export, so tests mock the instance they hand in. Request paths are relative to {@link BASE_URL},
+ * so matchers should use paths (`/stories/1`), not absolute URLs.
+ */
+export function mockHttp(): {http: AxiosInstance, mock: AxiosMockAdapter} {
```

ESLint

[error] Declare this without inline export and list it in a single export statement at the end of the file.

(yenz/export-at-end-of-file)